### PR TITLE
api validations completed

### PR DESCRIPTION
**Pull request by Mannyplg**

### CODE REVIEW

File: spec/requests/cats_request_spec.rb [hunk 1]
```diff
@@ -36,6 +36,67 @@
...
+      json = JSON.parse(response.body)
+      expect(json['enjoys']).to include "can't be blank"
+    end
+
+    it "does not create a cat without a image" do
+      cat_params = {
+        cat:{
+          name: 'Stephen',
+          age: 13,
+          enjoys: 'digging in the trees for bird nests',
```
Comment (WalkerTexas-coder) [inline spec/requests/cats_request_spec.rb]: does this break with the comma here?

File: spec/models/cat_spec.rb [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+    cat = Cat.create name:"Morgan", age: 5, image:"https://static.onecms.io/wp-content/uploads/sites/47/2020/11/30/cat-stuck-in-tree-298823311-2000.jpg"
+    expect(cat.errors[:enjoys]).to_not be_empty 
+  end
+
+  it "should validate presence of image" do
+    cat = Cat.create name:"Morgan", age: 5, enjoys:"Long walks"
+    expect(cat.errors[:image]).to_not be_empty 
+  end
+
+  it "should validate presence of enjoys" do
```
Comment (WalkerTexas-coder) [inline spec/models/cat_spec.rb]: This test name is for the other enjoys test. This one should be something like 
`it "should validate minimum length of enjoys"`

**Review by WalkerTexas-coder — COMMENTED** [review]
This is great! One change up and one question below. ALos it's stretch butt you are missing a Request test for enjoys minimum length if you want it.

File: spec/requests/cats_request_spec.rb (hunk 1 above)
Comment (Mannyplg) [reply spec/requests/cats_request_spec.rb]: Updated.

File: spec/models/cat_spec.rb (hunk 2 above)
Comment (Mannyplg) [reply spec/models/cat_spec.rb]: Updated...

**Review by WalkerTexas-coder — APPROVED** [review]
Awesome!
